docs/scenarios_authoring/Spel.md

```diff
@@ -259,7 +263,15 @@ numeric types and between `String` and some useful logical value types. Some exa
 | `'USD'`                                  | String     | Currency               |
 | `'bf3bb3e0-b359-4e18-95dd-1d89c7dc5135'` | String     | UUID                   |
 
-You can also use explicit conversions that are available in utility classes and build-in java conversion mechanisms:
+Usage example:
```

Comment:
To me this looks like a explicit conversion, not an implicit

Reply:
I would reduce amount of text with explanations (which I think is not necessary) by putting this into a  table with examples. The main reason is that we are very succinct in this document, relying more on examples rather than text. 

Column names would be "initial expression", "initial type", "explicit conversion", "target type", comment (if needed)

Your example would look like this (columns are comma-separated). 

#DATE.now  , Instant , #DATE.now.atZone(..) , ZonedDateTime, "your comment". 

Please note that types are capitalized. 

Having written the above, I remembered that there is a section on explicit date conversions in this document. Here is what I would do:

- add a link from the section we are commenting here to the section with "[conversions between date/time](There are basically three ways of storing date)"
- add information on how to obtain timestamp of type Instant with `#DATE.now` to the first bullet in "[Date/time data types"](https://nussknacker.io/documentation/docs/scenarios_authoring/Spel/#datetime-data-types).
- finally, I wonder whether it is correct to say _There are basically three ways of storing date_ if we enumerate five ways.

Reply:
I adjusted columns of suggested table. What do you think?